packages/material-ui/src/Select/SelectInput.js

```diff
@@ -281,7 +283,7 @@ class SelectInput extends React.Component {
           aria-pressed={open ? 'true' : 'false'}
           tabIndex={tabIndex}
           role="button"
-          aria-owns={open ? `menu-${name || ''}` : undefined}
+          aria-owns={open ? `menu-${name || ''}` : null}
```

Comment:
It should be undefined. Why changing it?

Reply:
That was totally unintended, I'll revert this change.